code-studio/src/js/code-studio.js

```diff
@@ -9,6 +9,9 @@
 'use strict';
 /* global Sounds */
 
+// Shim window.console to be safe in IE
+require('./consoleShim')(window);
```

Comment:
I'm checking in what I've got (solution B) and will consider cleaning this up into a wrapper layer (solution C) down the road.